ROLES.md

```diff
@@ -65,35 +65,23 @@ Further, you may self-nominate to be an approver by opening an issue with a list
         <p>Run their working group</p>
     </td>
     <td>Appointed by the <a href="./TECH-OVERSIGHT-COMMITTEE.md">technical oversight committee</a> as documented in <a href="./WORKING-GROUP-PROCESSES.md">Istio Working Group Processes</a></td>
-    <td>Write permissions on one or more repos, allowing issues to be manipulated.</td>
+    <td>Like a maintainer
   </tr>
 
   <tr>
     <td><a href="#administrator">Administrator</a></td>
     <td>Manage & control permissions</td>
     <td>Appointed by the <a href="./TECH-OVERSIGHT-COMMITTEE.md">technical oversight committee</a></td>
     <td>
-        <p>Admin privileges on the GitHub Istio org and all its repos</p>
-        <p>Admin privileges on the Istio Slack workspace</p>
-        <p>Admin privileges on the Istio Team Drive</p>
-        <p>Admin privileges on the Google Search Console for istio.io</p>
-        <p>Admin privilege to the Istio email lists.</p>
+        <p>Admin privileges on varous Istio-related resources, as defined in <a href="./ADMINS-FOR-ISTIO.md">ADMINS-FOR-ISTIO</a></p>
```

Comment:
This would be an informative statement, not sure if it's needed: admin privs are really orthogonal to being a maintainer:  admins are not automatically approvers which means they should not use their admin privs to merge anything that hasn't been approved by an approver.  That a rule that admins like myself and others have not always followed

Reply:
Added.